cmd/apprepository-controller/controller.go

```diff
@@ -619,3 +629,22 @@ func dbFlags() []string {
 		"--database-name=" + dbName,
 	}
 }
+
+// getImagePullSecretsRefs gets the []LocalObjectReference of Secrets from the
+// comma separated list passed in the repoSyncImagePullSecrets arg
```

Comment:
I think generally it's a good idea to parse the command-line input in the main.go (or call it from there) and then *pass* the result down, rather than referencing a global var from the command. So in this case, I'd be keen that we update the `NewController` function above to receive the `imagePullSecrets []corev1.LocalObjectReference`.

It's not essential, but when working on larger projects you'll find it gets very messy if different files are all just referencing global vars - generally it should be avoided.

Reply:
Also, whether you move this helper or not, can you pls add some unit tests. In particular, showing that spaces in the string don't result in `LocalObjectReference`'s with spaces.